docs_src/getting-started/Ch-GettingStartedHybrid.md

```diff
@@ -1,156 +1,80 @@
+
+1.  Get the service code
+    Per [Getting Started Go Developers](./Ch-GettingStartedGoDevelopers.md#Get-the-code), pull the micro service code you want to work on from GitHub. In
+    this example, we assume you want to get the device-virtual-go.
+    ``` bash
+    git clone https://github.com/edgexfoundry/device-virtual-go.git
+    ```
+2.  Build the service code
+    At this time, you can add or modify the code to make the service changes you need.  Once ready, you must compile and build the service into an executable.  Change folders to the cloned micro service directory and build the service.
+    ```
+    cd device-virtual-go/
+    make build
+    ```
+
+    ![image](EdgeX_GettingStartedHybridBuild.png)
+
+3.  Change the configuration
+    Depending on the service you are working on, you may need to change the configuration of the service to point to and use the other services that are containerized (running in Docker).  In particular, if the service you are working on is not on the same host as the Docker Engine running the containerized services, you will likely need to change the configuration.
+    Examine the configuration.toml file in the cmd/res folder of the device-virtual-go. Note that the Registry (located in the \[Registry\] section of the configuration) and all the "clients" (located in the \[clients\] section of the configuration file) suggest that the "Host" of these services is "localhost".  These and other host configuration elements need to change when the services are not running on the same host.  If you do have to change the configuration, save the configuration.toml file after making changes.
+
+4.  Run the service code natively.  The executable created by the make command is usally found in the cmd folder of the service.
+    ``` bash
+    cd cmd
+    ./device-virtual
+    ```
+
+    ![image](EdgeX_GettingStartedHybridRun.png)
+
+5.  Check the results
+    At this time, your virtual device micro service should be communicating with the other EdgeX micro services running in
+    their Docker containers. Give the virtual device a few seconds or so to
+    initialize itself and start sending data to Core Data. To check that it
+    is working properly, open a browser and point your browser to Core Data
+    to check that events are being deposited. You can do this by calling on
+    the Core Data API that checks the count of events in Core Data
+    http://[host].48080/api/v1/event/count.
+
+    ![image](EdgeX_GettingStartedHybridResults.png)
+
+!!! Note
+    If you choose, you can also import the service into GoLand and then code and run the service from GoLand.  Follow the instructions in the [Getting Started - Go Developers ](Ch-GettingStartedGoDevelopers#edgex-foundry-in-goland) to learn how to import, build and run a service in GoLand.
+
+    ![image](EdgeX_GettingStartedHybridGoLand.png)
```

Comment:
dont love the favoritsm of goland :P

Reply:
was there before so I just cleaned it up.  At the time we wrote the original material, GoLand was the tool everyone wanted to use but we found an amazing number of people unable to import the code to the tool.  

I am ok with either keeping or pulling it.

Reply:
I was just teasing 😀